test/format/config_def.c

```diff
@@ -102,12 +102,33 @@ CONFIG configuration_list[] = {
   {"checkpoint.wait", "seconds to wait if wiredtiger checkpoints configured",
     0x0, 5, 100, 3600, V_GLOBAL_CHECKPOINT_WAIT},
 
+  {"debug.checkpoint_retention", "adjust log removal to retain the log records",
+    0x0, 0, 128, 1024, V_GLOBAL_DEBUG_CHECKPOINT_RETENTION},
+
+  {"debug.cursor_copy", "make a copy of data returned by a cursor operation and return the copy using system allocator",
+    C_BOOL, 2, 0, 0, V_GLOBAL_DEBUG_CURSOR_COPY},
+
+  {"debug.eviction", "modify internal algorithms to force history store eviction to happen more aggressively",
+    C_BOOL, 2, 0, 0, V_GLOBAL_DEBUG_EVICTION},
+
+  {"debug.log_retention", "adjust log removal to retain at least this number of log files",
```

Comment:
What does the three numbers mean? 0, 128, 1024?

Reply:
The numbers 0, 128, 1024 were chosen based on the description in the config.sh file (min, maxrand, maxset) and the api data of the config